src/Core/NotificationHub/NotificationHubConnection.cs

```diff
@@ -95,6 +101,34 @@ public bool RegistrationEnabled(DateTime queryTime)
         return RegistrationStartDate < queryTime;
     }
 
+    public HttpRequestMessage CreateRequest(HttpMethod method, string pathUri, params string[] queryParameters)
+    {
+        var uriBuilder = new UriBuilder(Endpoint)
+        {
+            Scheme = "https",
+            Path = $"{HubName}/{pathUri.TrimStart('/')}",
+            Query = string.Join('&', [.. queryParameters, "api-version=2015-01"]),
```

Comment:
❓ Is this sort of thing and the related code due to the lack of SDK support?

Reply:
yes, we have to use the http interface